redux/slices/userSlice/UserSliceInitialState.Type.ts

```diff
@@ -19,4 +19,5 @@ export interface UserSliceInitialStateType {
       };
     };
   };
+  loading: "idle" | "pending" | "succeeded" | "failed";

```

Comment:
To bym przeniósł do enuma jako LoadingStates czy coś takiego i tutaj wrzucił jego nazwę. Może sięprzydać.